lib/hdb/hdb-ldap.c

```diff
@@ -357,7 +357,7 @@ LDAP_get_generalized_time_value(HDB * db, LDAPMessage * entry,
 				const char *attribute, KerberosTime * kt)
 {
     char *tmp, *gentime;
-    struct tm tm;
+    struct tm tm = {};
```

Comment:
Did a compiler complain about this?

Reply:
No, just something I noticed when I was debugging an unrelated issue.